ui/pages/asset/components/asset-v2.tsx

```diff
@@ -0,0 +1,368 @@
+                  <Text variant={TextVariant.headingMd} paddingBottom={4}>
+                    {t('marketDetails')}
+                  </Text>
+                  <Box
+                    display={Display.Flex}
+                    flexDirection={FlexDirection.Column}
+                    gap={2}
+                  >
+                    {marketData?.marketCap > 0 &&
+                      renderRow(
+                        t('marketCap'),
+                        <Text>
+                          {localizeLargeNumber(t, marketData.marketCap)}
+                        </Text>,
+                      )}
+                    {marketData?.totalVolume > 0 &&
+                      renderRow(
+                        t('totalVolume'),
+                        <Text>
+                          {localizeLargeNumber(t, marketData.totalVolume)}
+                        </Text>,
+                      )}
+                    {marketData?.circulatingSupply > 0 &&
+                      renderRow(
+                        t('circulatingSupply'),
+                        <Text>
+                          {localizeLargeNumber(t, marketData.circulatingSupply)}
+                        </Text>,
+                      )}
+                    {marketData?.allTimeHigh > 0 &&
+                      renderRow(
+                        t('allTimeHigh'),
+                        <Text>
+                          {formatCurrency(
+                            marketData.allTimeHigh,
+                            currency,
+                            getPricePrecision(marketData.allTimeHigh),
+                          )}
+                        </Text>,
+                      )}
+                    {marketData?.allTimeLow > 0 &&
+                      renderRow(
+                        t('allTimeLow'),
+                        <Text>
+                          {formatCurrency(
+                            marketData.allTimeLow,
+                            currency,
+                            getPricePrecision(marketData.allTimeLow),
+                          )}
+                        </Text>,
+                      )}
+                  </Box>
+                </Box>
+              )}
+              <Box marginBottom={8}>
+                <Text
+                  paddingLeft={4}
+                  paddingRight={4}
+                  variant={TextVariant.headingMd}
+                >
+                  {t('yourActivity')}
+                </Text>
+                {type === AssetType.native ? (
+                  <TransactionList hideTokenTransactions />
+                ) : (
+                  <TransactionList tokenAddress={address} />
+                )}
+              </Box>
+            </Box>
+          </Box>
+        </Box>
+        <Box
+          className="asset__footer"
+          backgroundColor={BackgroundColor.backgroundDefault}
+        >
+          <Box
+            marginLeft={'auto'}
+            marginRight={'auto'}
```

Comment:
These don't need `{}`

Reply:
Fixed in https://github.com/MetaMask/metamask-extension/pull/22508/commits/b8d898ae385d72eb86eb05ae9024260f6187dc21